### PR TITLE
Added Slack integration

### PR DESCRIPTION
**Pull request by bmispelon**
I realize this is quite a big PR but I oculd use some feedback on the copy (email, success/error pages, ...).

Thanks!

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,7 +15,10 @@ opbeat==3.3
 Pillow==3.2.0
 psycopg2==2.6.1
 pytz==2016.4
+requests==2.11.1
+slacker==0.9.0
 sqlparse==0.1.19
 static3==0.6.1
 tablib==0.11.2
+tomek==0.2
```
Comment (agagata) [inline requirements.txt]: ✨ 🎉

Comment (mjtamlyn) [inline requirements.txt]: 👯

File: slakslakslak/management/commands/send_invites.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class Command(BaseCommand):
+    help = 'Send pending invitation emails'
+
+    def handle(self, **kwargs):
+        pending = Invitation.objects.unsent()
+
+        msg = "This will send {} emails. Are you sure you want to continue? (y/N)".format(pending.count())
+        self.stdout.write(msg, ending=' ')
+        if input('').lower() not in {'y', 'yes', 'oui', 'ja', 'tak', '👍'}:
```
Comment (olasitarska) [inline slakslakslak/management/commands/send_invites.py]: 😂

File: slakslakslak/templates/slakslakslak/rules.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+<ol>
+    <li>Do not message people privately without their consent, and do not invite them
+to join any room without their consent.</li>
+    <li>Do not mention people who don't belong to particular room in that room.</li>
+    <li>Do not post advertizing or job postings outside of the dedicated #jobs channel.</li>
+</ol>
+
+<p>
+If at any time you're feeling uncomfortable due to anyone's behaviour on the Slack
+channel, please contact the organizers by emailing <a href="mailto:hello@djangounderthehood.com">hello@djangounderthehood.com</a>,
```
Comment (olasitarska) [inline slakslakslak/templates/slakslakslak/rules.html]: Should that be a coc@djangounderthehood.com email?

File: slakslakslak/forms.py [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        """
+        Restrict (mutate) the choices for the channels fields based on the
+        invitation object.
+        """
+        allowed = set(self.invitation.get_channels_from_ticket())
+        choices = self.fields['channels'].choices
+        self.fields['channels'].choices = [t for t in choices if t[0] in allowed]
+        print(self.fields['channels'].choices)
+
+    @tomek
```
Comment (oinopion) [inline slakslakslak/forms.py]: Yay!

**Pull request merged**